Examples/LambdaFunctions/scripts/deploy.sh

```diff
@@ -17,42 +17,25 @@ set -eu
 
 # Lambda Function name (must exist in AWS Lambda)
 lambda_name=SwiftSample
-
 # S3 bucket name to upload zip file (must exist in AWS S3)
 s3_bucket=swift-lambda-test
```

Comment:
As mentioned in some issue, we should make this bucket configurable since bucket names must be unique worldwide.

Reply:
we should prompt the user for the name of the bucket and lambda. is this what you are suggesting?

Reply:
yes. maybe store it between runs?

Reply:
added a prompt. wdyt?